tests/specs/benchmarks/verification.k

```diff
@@ -146,6 +146,14 @@ module VERIFICATION-HASKELL [symbolic, kore]
 
     rule maxUInt160 &Int Y => Y requires 0 <=Int Y andBool Y <=Int maxUInt160 [simplification]
 
+    rule ( #sizeByteArray(#ecrec( HASH, SIGV, SIGR, SIGS )) ==Int 0  ) =>           #ecrecEmpty ( HASH, SIGV, SIGR, SIGS )  [simplification]
+    rule ( #sizeByteArray(#ecrec( HASH, SIGV, SIGR, SIGS )) ==Int 32 ) => notBool ( #ecrecEmpty ( HASH, SIGV, SIGR, SIGS )) [simplification]
+
+    rule ( #sizeByteArray(#ecrec( _, _, _, _ )) <=Int K ) => true  
+      requires K >=Int 32 [simplification]
```

Comment:
```suggestion
      requires 32 <=Int K [simplification]
```

For performance (fewer rewrites). This repo tries hard to keep things using the `_<=Int_` and `_<Int_` operators where possible, both for readability (smaller numbers on the left), but also for not having to write as many lemmas.

This one probalby doesn't matter, since `K` will likely always be 0, 20, or 32. But just something to note.